addons/mail/models/mail_composer_mixin.py

```diff
@@ -22,7 +22,7 @@ class MailComposerMixin(models.AbstractModel):
 
     # Content
     subject = fields.Char('Subject', compute='_compute_subject', readonly=False, store=True)
-    body = fields.Html('Contents', sanitize_style=True, compute='_compute_body', store=True, readonly=False)
+    body = fields.Html('Contents', compute='_compute_body', store=True, readonly=False, sanitize=False)
```

Comment:
C'est normal qu'on ne sanitize plus ici ?

Reply:
Yes because now if we sanitize, we cannot use dynamic content (`t-if`/`t-out`, ...).
Before it was possible while sanitizing because the dynamic expression of jinja were not sanitized.